### PR TITLE
feat(preset-mini): add rawColorComment

### PR DESCRIPTION
**Pull request by Simon-He95**
close: #3816

### CODE REVIEW

**Comment by zyyv** [comment]
@Simon-He95  I think this option is more integrated in the vscode plugin and has nothing to do with the `core` package, and has nothing to do with parsing the output css.

**Comment by Simon-He95** [comment]
> I think this option is more integrated in the vscode plugin and has nothing to do with the `core` package, and has nothing to do with parsing the output css.

Because it is difficult to disassemble, a field that is not used by the client is added to process it. This field is actually only provided for the vscode side.

File: packages/core/src/types.ts [hunk 1]
```diff
@@ -916,4 +920,9 @@ export interface GenerateOptions<T extends boolean> {
    * If return extended "matched" with payload and count
    */
   extendedInfo?: T
+
+  /**
+   * show origin theme color
+   */
+  showOriginThemeColor?: boolean
```
Comment (antfu) [inline packages/core/src/types.ts:927]: We probably don't need a new option for that. We could always do that when `envMode` is `dev`

**Comment by antfu** [comment]
Werid that the test if failing

**Pull request closed without merging**